Sources/PaymentsCore/Networking/GraphQL/Common/Models/GraphQLQueryResponse.swift

```diff
@@ -0,0 +1,5 @@
+
+
+internal struct GraphQLQueryResponse<T: Codable>: Codable {
+    let data: T?
+}
```

Comment:
Will more properties be added to this struct? Also we don't need `internal`, since it's the default.

Reply:
Yeah all GraphQL responses have a top-level `data` property unfortunately.